api/v1_queues.inc

```diff
@@ -0,0 +1,125 @@
+<?php
+include_once($relPath.'Project.inc');
+include_once($relPath.'release_queue.inc');
+include_once($relPath.'user_is.inc');
+include_once("exceptions.inc");
+include_once("misc.inc");
+
+// DP API v1 -- Queues
+
+//---------------------------------------------------------------------------
+// queues -- list queues, optionally filtered by round
+function api_v1_queues($method, $data, $query_params)
+{
+    global $Round_for_round_id_;
+
+    $roundid = array_get($query_params, "roundid", null);
+    if (!is_null($roundid)) {
+        try {
```

Comment:
FWIW, you can use `validate_round()` here:
```php
$round = validate_round($roundid, null);
```

Reply:
Done